store/models/common.go

```diff
@@ -21,21 +21,21 @@ const (
 	RunStatusInProgress = RunStatus("in progress")
 	// RunStatusPendingConfirmations is used for when a run is awaiting for block confirmations.
 	RunStatusPendingConfirmations = RunStatus("pending_confirmations")
-	// RunStatusPendingExternal is used for when a run is waiting on the completion
+	// RunStatusPendingBridge is used for when a run is waiting on the completion
 	// of another event.
-	RunStatusPendingExternal = RunStatus("pending_external")
+	RunStatusPendingBridge = RunStatus("pending_bridge")
```

Comment:
Definitely prefer these names, as the only part of the system that should know about anything "external" are the bridges.